dbcon/execplan/calpontselectexecutionplan.cpp

```diff
@@ -100,12 +100,13 @@ CalpontSelectExecutionPlan::CalpontSelectExecutionPlan(const int location)
 
 CalpontSelectExecutionPlan::CalpontSelectExecutionPlan(
     const ReturnedColumnList& returnedCols, ParseTree* filters, const SelectList& subSelects,
-    const GroupByColumnList& groupByCols, ParseTree* having, const OrderByColumnList& orderByCols,
+    const GroupByColumnList& groupByCols, uint32_t groupByKeysCount, ParseTree* having, const OrderByColumnList& orderByCols,
     const string alias, const int location, const bool dependent, const bool withRollup)
  : fReturnedCols(returnedCols)
  , fFilters(filters)
  , fSubSelects(subSelects)
  , fGroupByCols(groupByCols)
```

Comment:
Is there a difference b/w `groupByCols` and introduced `groupByKeysCount`?

Reply:
Yes. It is explained in the comment next to groupByKeysCount() functions of CSEP.

Reply:
Makes sense.